src/consensus.md

```diff
@@ -0,0 +1,85 @@
+* **Document author**: The person who authored the decision document. They ultimately decide what changes they wish to make to the text in response to concerns. They often do this in close consultation with a member of the lang team, but that is not required.
+
+## The process
+
+Consensus decisions are done using rfcbot. They always begin with a decision document authored by the [document author](#definitions). The document author can be anyone, they do not have to be a member of a Rust team.
+
+To make a decision, a [lang team member or advisor](#definitions) should issue `@rfcbot fcp merge`. This will create checkboxes. Once [2/3 of the boxes have been checked](#doing-the-math), the decision will enter final-comment-period. During that period the team should review any comments raised — assuming no concerns are raised (see below), the decision is finalized once the FCP has expired.
+
+*Note:* Before finalizing the decision, the team should engage with **new** points raised on the thread (particularly if they are made by people not able to raise formal concerns).
+
+## Raising concerns
+
+[Lang team members or advisors](#definitions) are entitled to raise a **formal concern** during the discussion process. This can be done with `@rfcbot concern concern-name`. The decision cannot be finalized until the concern is resolved, either because the person who raised the concern is satisfied and [resolves](#resolving-a-concern) it themselves, or because the team decides to [challenge it](#challenging-a-concern).
+
+### Expectations on raising a concern
+
+The [lang team member or advisor](#definitions) who raises a concern is expected to
+
+* write a constructive comment explaining the concern;
+* make themselves available for discussion in a reasonable fashion;
+* be prepared to write a document summarizing their concern for the lang-team if requested.
+
+### When a concern is raised
+
+When a concern is raised, the team will review the concern in the triage meeting. Based on that discussion, the [document author](#definitions) can decide whether they wish to make changes to mitigate or resolve the concern.
+
+## Resolving a concern
+
+If the person who raised the concern is satisfied, either because of changes made or because they don't wish to continue blocking progress, they should resolve the concern themselves with `@rfcbot resolve`. This is the desired outcome.
+
+If the concern has still not been resolved, there will typically be a period of deeper discussion. The goal is to [find common ground](./making_decisions.md#design-axioms), either by mitigating it directly or by side-stepping it via a subset of the design that still achieves the major goals.
+
+### Challenging a concern
+
+If deeper discussion does not reach a solution, a [lang team member](#definitions) can challenge a concern. This indicates that they feel they understand the concern and do not agree it is a problem.
```

Comment:
```suggestion
If deeper discussion does not reach a solution, a [lang team member or advisor](#definitions) can challenge a concern. This indicates that they feel they understand the concern and do not agree it is a problem.
```

I think anyone who can champion should be able to challenge a concern.

Reply:
In a parliamentary sense, I think this challenge is equivalent to proposing a motion, and so I'd say it should be anyone who can propose FCP merge, which is currently only team members.